libyul/backends/evm/AbstractAssembly.h

```diff
@@ -98,7 +99,7 @@ class AbstractAssembly
 	/// Append the assembled size as a constant.
 	virtual void appendAssemblySize() = 0;
 	/// Creates a new sub-assembly, which can be referenced using dataSize and dataOffset.
-	virtual std::pair<std::shared_ptr<AbstractAssembly>, SubID> createSubAssembly(bool _creation, std::string _name = "") = 0;
+	virtual std::pair<std::shared_ptr<AbstractAssembly>, SubID> createSubAssembly(langutil::EVMVersion _evmVersion, bool _creation, std::string _name = "") = 0;
```

Comment:
A sub-assembly will always have to have the same evm-version as the parent assembly, which you should have available in both implementations of this, shouldn't you?

Reply:
Yes, that is a good point. I believe I did this change before I had added `evmVersion` as a member of `evmasm::Assembly`. I will do the changes.